.changeset/chilled-files-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': minor
+---
+
+Added methods to the Apps-Engine that interact with unread messages
```

Comment:
Changesets should be in present tense

(i didnt change this but somehow im the one enforcing it lel)